ROSCO/src/ROSCO_Types.f90

```diff
@@ -126,6 +126,13 @@ MODULE ROSCO_Types
     REAL(DbKi)                    :: PC_RtTq99                   ! 99% of the rated torque value, using for switching between pitch and torque control, [Nm].
     REAL(DbKi)                    :: VS_MaxOMTq                  ! Maximum torque at the end of the below-rated region 2, [Nm]
     REAL(DbKi)                    :: VS_MinOMTq                  ! Minimum torque at the beginning of the below-rated region 2, [Nm]
+    INTEGER(4)                    :: LIDAR_ControlMode                   ! Flag of lidar assisted Pitch forward control
```

Comment:
Please add these new `LocalVars` to the [`rosco_registry.yaml`](https://github.com/NREL/ROSCO/blob/main/ROSCO/rosco_registry/rosco_types.yaml) and re-run [`write_registry.py`](https://github.com/NREL/ROSCO/blob/main/ROSCO/rosco_registry/write_registry.py).  It will overwrite this file, so please be careful or save a copy.

Reply:
I've added the LocalVars and CntrPars to the rosco_registry.yaml. I had to create a new derived type called LidarVariables for the lidar measurement buffers because otherwise it would add all 2000 entries to ROSCO_IO.F90.